CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,40 @@
+# CLX Code of Conduct
+
+CLX has adopted the [Contributor Covenant Conde of Conduct](https://docs.rapids.ai/resources/conduct):
+
+## Our Pledge
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+### Examples of behavior that contributes to creating a positive environment include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+- Examples of unacceptable behavior by participants include:
+
+### The use of sexualized language or imagery and unwelcome sexual attention or advances
```

Comment:
It seems like this is instead a list item and the line above `Examples of unacceptable behavior by participants include:`  is the list header?